src/utilities/utility.ts

```diff
@@ -129,6 +138,54 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getDeploymentConfig(): Promise<DeploymentConfig> {
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];

```

Comment:
@ds-ms suggested that we use `;` as a delimiter. `\n` doesn't work while setting variables.
Good to `trim` the extracted path. (Using `map` after filter?)

Reply:
Done